### PR TITLE
correct rendering utf-8 IFrame (again)

### PR DESCRIPTION
**Pull request by knil-sama**
Hi,

I was too hasty last time and only update one of the two line that need the "charset:utf-8"
So here the full version + a dedicated test for IFrame element including rendering.
But for this test phantomJs is required and it's not installable using pip (selenium also but I simply add it in requirement-dev.txt).
And I am not  sure if this is worth breaking automatic installation for this.

### CODE REVIEW

File: requirements-dev.txt [hunk 1]
```diff
@@ -1,3 +1,4 @@
 flake8
 pytest
 vincent
+selenium
```
Comment (ocefpaf) [inline requirements-dev.txt]: :+1:

**Comment by ocefpaf** [comment]
@knil-sama can you take a look at the failing test so we can merge this?

**Comment by knil-sama** [comment]
@ocefpaf The issue is with conda because you need to input the channel to use for install for some package like selenium, 
in this case "conda install -c bokeh selenium" work fine but for some reason the selenium line from conda list don't,
I need to look a little more into this, after that I should find a way that work both with conda and pip for installation.

**Comment by ocefpaf** [comment]
I need to update the Travis-CI configuration there. Let me do that first and I may help you with this. Hold on...

**Comment by ocefpaf** [comment]
@knil-sama can you rebase with latest master and check if this will pass?

**Comment by knil-sama** [comment]
@ocefpafI will do this

File: tests/test_iframe.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+""""
+Folium Element Module class IFrame
+----------------------
+"""
+import branca.element as elem
+from selenium import webdriver
+import base64
```
Comment (ocefpaf) [inline tests/test_iframe.py]: There are a few coding standards issues here. From the Travis-CI logs:

```
./tests/test_iframe.py:8:1: F401 'base64' imported but unused
./tests/test_iframe.py:9:1: E302 expected 2 blank lines, found 0
./tests/test_iframe.py:10:1: W191 indentation contains tabs
./tests/test_iframe.py:11:1: W191 indentation contains tabs
./tests/test_iframe.py:13:1: E302 expected 2 blank lines, found 1
./tests/test_iframe.py:14:1: W191 indentation contains tabs
./tests/test_iframe.py:14:48: E231 missing whitespace after ','
./tests/test_iframe.py:14:57: E231 missing whitespace after ','
./tests/test_iframe.py:15:1: W191 indentation contains tabs
./tests/test_iframe.py:17:1: E302 expected 2 blank lines, found 1
./tests/test_iframe.py:18:1: W191 indentation contains tabs
./tests/test_iframe.py:19:1: W191 indentation contains tabs
./tests/test_iframe.py:20:1: W191 indentation contains tabs
./tests/test_iframe.py:21:1: W191 indentation contains tabs
./tests/test_iframe.py:21:27: E703 statement ends with a semicolon
./tests/test_iframe.py:22:1: W191 indentation contains tabs
```

Those should be fairly easy to fix. Thanks for you patience @knil-sama! We are almost ready to merge this!

Comment (knil-sama) [inline tests/test_iframe.py]: done, during your review ;)

**Comment by ocefpaf** [comment]
Thanks for the fix @knil-sama!

**Comment by knil-sama** [comment]
Thanks for you work too @ocefpaf  👍

**Pull request merged**